contracts/collectxyz-nft-contract/schema/xyz_tokens_response.json

```diff
@@ -67,7 +67,7 @@
     },
     "Expiration": {
       "description": "Expiration represents a point in time when some event happens. It can compare with a BlockInfo and will return is_expired() == true once the condition is hit (and for every block in the future)",
-      "anyOf": [
+      "oneOf": [
```

Comment:
Is this schema something you generate manually?

Reply:
It's generated by running `cargo schema` from the NFT contract package root directory. The `schema` command executes this file: https://github.com/collectxyz/collectxyz-nft-contract/blob/0xja/crate-release-prep/contracts/collectxyz-nft-contract/examples/schema.rs